pkg/crc/config/secret_config.go

```diff
@@ -0,0 +1,69 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/spf13/cast"
+	"github.com/zalando/go-keyring"
+)
+
+const CRCSecretServiceName = "crc"
+
+var ErrSecretsNotAccesible = errors.New("secret store is not accesible")
```

Comment:
"Accessible" takes 2 's'. This happens multiple times in this file.

Reply:
fixed now, thanks :)